fetch.bs

```diff
@@ -7803,11 +7802,12 @@ constructor steps are:
 
  <li><p>Set <a>this</a>'s <a for=Request>request</a> to <var>request</var>.
 
- <li><p>Set <a>this</a>'s <a for=Request>signal</a> to a <a for=/>new</a> {{AbortSignal}} object
- with <a>this</a>'s <a>relevant Realm</a>.
+ <li><p>Let <var>signals</var> be « <var>signal</var> » if <var>signal</var> is non-null; otherwise
+ « ».
 
- <li><p>If <var>signal</var> is non-null, then make <a>this</a>'s <a for=Request>signal</a>
- <a for=AbortSignal>follow</a> <var>signal</var>.
+ <li><p>Set <a>this</a>'s <a for=Request>signal</a> to the result of
+ <a>creating a new composite abort signal</a> from <var>signals</var>, using {{AbortSignal}} and
```

Comment:
"creating a composite abort signal" (or we can rename the algo to add "new"). I left off "new" since I thought it was implied by "create". Same below.